### PR TITLE
EZP-25274: Update README of ezplatform-demo repository

### PR DESCRIPTION
**Pull request by clash82**
Link to Jira task: https://jira.ez.no/browse/EZP-25284

Image included in README:

demo

### CODE REVIEW

**Comment by clash82** [comment]
ping @SylvainGuittard @DominikaEZ

File: README.md [hunk 1]
```diff
@@ -6,66 +6,77 @@
...
+## What is eZ Platform Demo ?
+
+*eZ Platform Demo* is an example of a simple website using *eZ Platform*. This example should not be use for starting a new project.
+
+Features:
+- Blog
+- Gallery
+- Contact Form
 
 ## Requirements
```
Comment (DominikaEZ) [inline README.md]: Current requirements: https://doc.ez.no/display/TECHDOC/Requirements

**Comment by SylvainGuittard** [comment]
Could you please add a screenshot?

**Comment by clash82** [comment]
@SylvainGuittard Yes, as soon as I will fix my Apache...

**Comment by SylvainGuittard** [comment]
@clash82 ;)

File: README.md (hunk 1 above)
Comment (SylvainGuittard) [inline README.md]: Good point @DominikaEZ 
Thanks

Comment (clash82) [inline README.md]: @DominikaEZ can I link to this page directly or the URL may change?

Comment (DominikaEZ) [inline README.md]: I don't expect it to change. If it should, I'll notify you.

Comment (clash82) [inline README.md]: Thank you :)

File: README.md [hunk 2]
```diff
@@ -6,66 +6,77 @@
...
+Clone the Github repository containing the demo:
+
+```bash
+git clone https://github.com/ezsystems/ezplatform-demo
+cd ezplatform-demo
+```
+
+### 3. Install dependencies
+
+Run `composer install` to install the required dependencies:
```
Comment (andrerom) [inline README.md]: Should be composer install I think, or 2 and 3 can be combined by using composer create-project once this repo exists on packagist.

Comment (clash82) [inline README.md]: Yes, of course. Fixed.

**Comment by SylvainGuittard** [comment]
+1

**Pull request merged**